packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/WrongNetworkDropdown.tsx

```diff
@@ -20,7 +21,10 @@ export const WrongNetworkDropdown = () => {
           <button
             className="menu-item text-error btn-sm !rounded-xl flex gap-3 py-3"
             type="button"
-            onClick={() => disconnect()}
+            onClick={async () => {
```

Comment:
couldn't reproduce this with walletConnect, changed just to be safe. + added `signOut()`